docs/modules/extend/examples/pdf-converter-article-title-with-author-and-date.rb

```diff
@@ -0,0 +1,43 @@
+class PDFConverterArticleTitleWithAuthorAndDate < (Asciidoctor::Converter.for 'pdf')
+  register_for 'pdf'
+
+  def ink_general_heading doc, title, opts
+    return super unless opts[:role] == :doctitle # <1>
+    ink_document_title title, opts # <2>
+    ink_document_metainfo doc
+    ink_titlesection_body_separator
+  end
+
+  def ink_document_title title, opts
+    if (top_margin = @theme.heading_h1_margin_page_top || @theme.heading_margin_page_top) > 0
+      move_down top_margin
+    end
+    pad_box @theme.heading_h1_padding do
+      if (transform = resolve_text_transform opts)
+        title = transform_text title, transform
+      end
+      if (inherited = apply_text_decoration font_styles, :heading, 1).empty?
+        inline_format_opts = true
+      else
+        inline_format_opts = [{ inherited: inherited }]
+      end
+      typeset_text_opts = { color: @font_color, inline_format: inline_format_opts }.merge opts
+      typeset_text title, (calc_line_metrics (opts.delete :line_height) || @base_line_height), typeset_text_opts 
+    end
+  end
+
+  def ink_document_metainfo doc
+    revremark = doc.attr 'revremark' # <3>
+    if doc.author || doc.revdate || revremark # <4>
+      theme_font :base do
+        author_date_separator = (doc.author && doc.revdate) ? ' – ' : '' # <5>
```

Comment:
Because `&&` has higher precedence than `… ? … : …`, the braces are now redundant. Should they be removed like in https://github.com/asciidoctor/asciidoctor-pdf/pull/2439#issuecomment-1748391437 or be left in for those not aware of Ruby's precedence rules?

Reply:
I prefer to use as few braces as possible. So if it works without them, then drop them. That's the rule of thumb I follow.

Reply:
removed redundant braces with f66cef0058baf2e9cd1fc696c2bd4b1dca2070b2